thyrel-web/src/components/room/start/SentenceForm.tsx

```diff
@@ -42,6 +42,14 @@ export default function StartForm() {
       : setSentence(e.target.value);
   }
 
+  function handleKeyPress(e: React.KeyboardEvent<HTMLInputElement>) {
+    if (loading || e.key !== 'Enter') {
+      return;
+    }
```

Comment:
si tu veux pour faire plus cours, y'a certaine personne qui **n'aime pas du tout** cette synaxe, perso pour un `return` tout simple ça me choque pas!
```suggestion
    if (loading || e.key !== 'Enter') return;
```